code/game/gamemodes/cult/cult_comms.dm

```diff
@@ -65,3 +69,144 @@
 	popup.set_content(text)
 	popup.open()
 	return 1
+
+/mob/living/proc/cult_master()
+	set category = "Cultist"
+	set name = "Assert Leadership"
+	pollCultists(src)
+
+/datum/action/innate/cultmast/IsAvailable()
+	if(owner.mind.special_role != "Cult Master")
+		return 0
+	return ..()
+
+/datum/action/innate/cultmast/finalreck
+	name = "Final Reckoning"
+	desc = "A single use spell that brings the entire cult to the master's location"
+	button_icon_state = "sintouch"
+	background_icon_state = "bg_demon"
+	buttontooltipstyle = "cult"
+	check_flags = AB_CHECK_RESTRAINED|AB_CHECK_STUNNED|AB_CHECK_CONSCIOUS
+
+/datum/action/innate/cultmast/finalreck/Activate()
+	var/list/destinations = list()
+	for(var/turf/T in orange(1,owner))
+		destinations += T
+	for(var/i in 1 to 4)
+		if(do_after(owner, 30, target = owner))
```

Comment:
Its like all the clock cult spells where the channeling is broken up into parts for effect.